src/python/espressomd/profiler.pyx

```diff
@@ -0,0 +1,31 @@
+cimport c_profiler
+
+
+def begin_section(name):
+    """
+    Start named section in profiler.
+
+    Parameters
+    ----------
+
+    name: obj:`str`
+        Name of the section
+
+    """
+
+    c_profiler.begin_section(name)
```

Comment:
should we check if the profiler is available?
```
IF NOT HAVE_CALIPER:
    raise ValueError("No profiler available.")
```

Reply:
It does nothing if it's not available, which is fine.